rsmtool/container.py

```diff
@@ -260,6 +302,50 @@ def __iter__(self):
         for key in self.keys():
             yield key
 
+    def drop(self, name):
+        """
+        Drop a given data frame from the
+        container.
+
+        Parameters
+        ----------
+        name : str
+            The name of the data frame to drop from the
+            container object.
+
+        Returns
+        -------
+        self
+        """
+        if name in self._names:
+            self._names.remove(name)
+            self._dataframes.pop(name)
+            self._data_paths.pop(name)
+        return self
+
+    def rename(self, name, new_name):
+        """
+        Rename a given data frame in the
+        container.
+
+        Parameters
+        ----------
+        name : str
+            The name of the current data frame
+            in the container object.
+        new_name : str
+            The the new name for the data frame
+            in the container object.
+
+        Returns
+        -------
+        self
+        """
+        df = self._dataframes[name]
```

Comment:
Do we also need to update the `data_path` is that's specified in the container?

Reply:
Yes, we should do that.